### ISSUE TITLE
E - Build a QC matrix - Do we need this in the Query and View Environment?

### ISSUE THREAD

**Issue by James-Dargie** [jncc/marine-recorder#44]
Do we need this in the Query and View environment? I'd assume those doing QA would be working in the Create, Edit and Publish Environment? That said if people think the ability to generate matrices for purposes other than QA I have no objection to such a tool in the Q&V environment  - perhaps it is just a naming thing...

**Comment by GMDuncan** [comment]
Agreed that it is less fundamental to the Q+V environment, it was added there as a "low-cost" improvement as the page functionality would be identical, (only a change in source data).

Happy to be considered now or at development depending on costs.
